esmvalcore/_data_finder.py

```diff
@@ -484,17 +483,33 @@ def get_output_file(variable, preproc_dir):
     return outfile
 
 
-def get_statistic_output_file(variable, preproc_dir):
-    """Get multi model statistic filename depending on settings."""
-    updated_var = copy.deepcopy(variable)
-    updated_var['timerange'] = updated_var['timerange'].replace('/', '-')
-    template = os.path.join(
+def get_multiproduct_filename(attributes, preproc_dir):
+    """Get ensemble/multi-model filename depending on settings."""
+    relevant_keys = [
+        'project', 'dataset', 'exp', 'ensemble_statistics',
+        'multi_model_statistics', 'mip', 'short_name'
+    ]
+
```

Comment:
do we really want to allow for inter-project grouping? This is tricky: CMIP5+CMIP6 maybe makes some sense, but CMIP+OBS?

Reply:
You can exclude the obs in the `exclude` parameter

Reply:
cheers, do you have a clear example for the use of `exclude`?

Reply:
Just added a test

Reply:
excellent, it'd be good to have one in the preprocessor example recipe - @schlunma is putting one together